docs/source/DeveloperGuide/Systems/Framework/stableconns/service.py

```diff
@@ -1,15 +1,18 @@
-import types
-from DIRAC import S_OK, S_ERROR
+""" example
```

Comment:
Shall we put here a meaning full example of docstring, with the configuration options, that will then be generated, as discussed at the DUW ?